### PR TITLE
feat: Add ability to store VC in user agent wasm

### LINKED ISSUE #24: Add ability to store VC in user agent wasm

### PR DESCRIPTION
**Pull request by fqutishat**
closes #24

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: pkg/internal/mock/storage/mock_store.go [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+	s.lock.RLock()
+	defer s.lock.RUnlock()
+
+	val, ok := s.Store[k]
+	if !ok {
+		return nil, storage.ErrDataNotFound
+	}
+
+	return val, s.ErrGet
```
Comment (sandrask) [inline pkg/internal/mock/storage/mock_store.go]: return val, nil

Comment (fqutishat) [reply pkg/internal/mock/storage/mock_store.go]: done

File: cmd/user-agent/web/src/views/StoreVC.vue [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+</template>
+
+<script>
+    import * as polyfill from "credential-handler-polyfill";
+    import * as webCredentialHandler from "web-credential-handler";
+
+    async function handleWalletReceiveEvent() {
+        const credentialEvent = await webCredentialHandler.receiveCredentialEvent();
+        const vcData = credentialEvent.credential.data
+        window.console.log('Received event:', vcData);
```
Comment (troyronda) [inline cmd/user-agent/web/src/views/StoreVC.vue]: remove

Comment (fqutishat) [reply cmd/user-agent/web/src/views/StoreVC.vue]: I will remove in the next PR

**Pull request merged**